test.js

```diff
@@ -152,9 +152,11 @@ test ( '/* rtl:ignore */ can be used inside /* rtl:begin:ignore */ and /* rtl:en
 .foo { padding-left: 0 }
 /* rtl:ignore */
 .bar { direction: ltr }
+.baz { left: 0 }
```

Comment:
before this fix, this selector would be RTL'ed, even though it is still inside a begin/ignore block